variables.tf

```diff
@@ -3282,3 +3294,15 @@ variable "use_app_repo_for_cd_deploy" {
   description = "Set to `true` to use the CI sample application repository as the deployment repository in the CD pipeline. This will be set in the pipeline config integration."
   default     = false
 }
+
+variable "service_name_pipeline" {
+  type        = string
+  description = "The name of the Service ID for pipeline and toolchain access."
+  default     = "alm-pipeline-service-id"
```

Comment:
can we call this` toolchain-pipeline-service-id` , want to remove alm as names of the DA's are changing